src/fragmentTypesGenerator.js

```diff
@@ -0,0 +1,36 @@
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+fetch('http://localhost:4000/graphql', {
```

Comment:
instead of localhost, can we do: `process.env.GRAPHQL_URI || 'http://localhost:4000/graphql'`? that way it will also work when deploying to prod.